src/test/e2e/api/admin/api-token.auth.e2e.test.ts

```diff
@@ -51,18 +52,27 @@ test('editor users should only get client or frontend tokens', async () => {
     const { request, destroy } = await setupAppWithCustomAuth(stores, preHook);
 
     await stores.apiTokenStore.insert({
+        environment: '',
+        projects: [],
+        tokenName: '',
```

Comment:
I guess this is fine, but unsure if this ends up being the same as sending undefined. I trust it because the test fails, but I'm not sure if we're actually testing the output of this insert